pkg/descheduler/strategies/duplicates.go

```diff
@@ -50,6 +51,10 @@ func validateRemoveDuplicatePodsParams(params *api.StrategyParameters) error {
 	return nil
 }
 
+type podOwner struct {
```

Comment:
could you just use `metav1.OwnerReference` for this?

Reply:
`metav1.OwnerReference` does not have a namespace field

Reply:
It does have a `uid` though, which should be similarly unique right?